packages/auth-construct/src/construct.ts

```diff
@@ -885,7 +885,7 @@ export class AmplifyAuth
         output.oauthRedirectSignOut = this.oAuthSettings.logoutUrls
           ? this.oAuthSettings.logoutUrls.join(',')
           : '';
-        output.webClientId = this.resources.userPoolClient.userPoolClientId;
+        output.oauthClientId = this.resources.userPoolClient.userPoolClientId;
```

Comment:
This fixes a bug where oauthClientId was not being set, .
The related test has been updated to expect oauthClientId.